.github/workflows/release.yml

```diff
@@ -178,6 +178,19 @@ jobs:
           file: _dist/spin-${{ env.RELEASE_VERSION }}-${{ env.RUNNER_OS }}-${{ matrix.config.arch }}.zip
           tag: ${{ github.ref }}
 
+      - name: Configure AWS Credentials
+        if: runner.os == 'linux' && matrix.config.arch == 'amd64'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.INFRA_NAMESPACE }}-${{ secrets.AWS_REGION }}-spin-release-artifacts
```

Comment:
```suggestion
          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/spin-release-artifacts-${{ secrets.AWS_REGION }}
```

Reply:
✅, dropped region also from the role name